eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/location/googleapiclient/BasicGoogleApiClientHolder.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.location.googleapiclient;
+
+import android.app.Activity;
+
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.schedjoules.eventdiscovery.framework.common.BaseActivity;
+
+
+/**
+ * Basic implementation for {@link GoogleApiClientHolder}, it configures the client to be auto-managed.
+ *
+ * @author Gabor Keszthelyi
+ */
+public class BasicGoogleApiClientHolder implements GoogleApiClientHolder
```

Comment:
Added all missing final modifiers in the project. (Will update the project file template later)